app/Platform/RouteServiceProvider.php

```diff
@@ -59,6 +60,7 @@ protected function mapWebRoutes(): void
             // Route::get('game/{game}/assets', [GameAssetsController::class, 'index'])->name('game.asset.index');
             // Route::get('game/{game}/players', [GamePlayerController::class, 'index'])->name('game.player.index');
             Route::get('game/{game}/dev-interest', GameDevInterestController::class)->name('game.dev-interest');
+            Route::get('game/{game}/hashes/manage', ManageHashesController::class)->name('game.manage-hashes');
```

Comment:
I did quite a bit of research on this and landed somewhere other than `index`.

It seems the standard conventions are mirroring RESTful architecture. The manage hashes page is somewhere at the boundary of that, largely because it allows the user to see multiple hashes (so it doesn't fit neatly with `::edit()`) but only manage one per API call.

Based on what I've studied, `index` is more appropriate for the "Supported Game Files" page.
`edit` would be appropriate if the page only showed a single resource for editing.
In the absence of a standard conventional name, one can be invented that is most appropriate for the use case; I chose `manage` and changed the name to "game.hashes.manage".

Reply:
For resources and sub-resources in route names (not the URL, just the name) i think all the others are singular: `game.hash.manage`